From f31ae8cbc1768c351d0f1803b91d39b259dcda5e Mon Sep 17 00:00:00 2001
From: Carsten Hansen <Carsten.Hansen@bksv.com>
Date: Tue, 23 Apr 2019 14:59:15 +0200
Subject: [PATCH] Prevent use-after-free when specifying adv timeout of 0

---
 src/advertising.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/advertising.c b/src/advertising.c
index 625750c..37bde65 100644
--- a/src/advertising.c
+++ b/src/advertising.c
@@ -582,8 +582,10 @@ static bool parse_timeout(DBusMessageIter *iter,
 	if (client->to_id)
 		g_source_remove(client->to_id);
 
-	client->to_id = g_timeout_add_seconds(client->timeout, client_timeout,
-								client);
+	if (client->timeout > 0) {
+		client->to_id = g_timeout_add_seconds(client->timeout, client_timeout,
+									client);
+	}
 
 	return true;
 }
